### DISCUSSION TITLE
use system libraries (qt5)

### DISCUSSION THREAD

**Discussion by Kicer86** [microsoft/vcpkg#17390] [Q&A]
Hi

I use vcpkg to install all dependencies for my app and one of them is Qt5.
It would be very convenient for me if I could force vcpkg to use Qt installed on my computer
instead of downloading and building Qt from ports. The main reason of this is that 
I use appveyor and travis-ci for testing builds and it would take too long to build Qt there (both services have time limits for jobs).
My idea is to use --overlay-ports option to point to a directory with my custom Qt scripts which would somehow 'import' system Qt into vcpkg environment. As far as I understand this kind of trick is done for cuda port. Is this way of doing things officially supported and documented (I could not find anything about it) or is it rather a dirty hacky workaround?

**Comment by dg0yt** [comment]
I don't know what is official answer but I think that overlay ports is what would be proposed here.

Maybe there is a demand for a collection of frequently used overlay ports, for example for usage with vendor-provided Qt. Or some vendors might provide overlay ports with their packages.

**Comment by hmoffatt** [comment]
This is also helpful for Qt when not using Qt open source.
